File: app/src/main/java/shaishav/com/bebetter/data/models/Goal.java
package shaishav.com.bebetter.data.models;


/**
 * Created by Shaishav on 9/5/2016.
 */
public class Goal {

    long id, date, goal;

    String server_id;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getDate() {
        return date;
    }

    public void setDate(long date) {
        this.date = date;
    }

    public long getGoal() {
        return goal;
    }

    public void setGoal(long goal) {
        this.goal = goal;
    }

    public String getServer_id() {
        return server_id;
    }

    public void setServer_id(String server_id) {
        this.server_id = server_id;
    }
}

File: app/src/main/java/shaishav/com/bebetter/data/models/Goal.kt
/*
 * Copyright (c) 2018 Shaishav Gandhi
 *  Licensed under the Apache License, Version 2.0 (the "License");
 *  You may not use this file except in compliance with the License.
 *  You may obtain a copy of the License at
 *
 *  http://www.apache.org/licenses/LICENSE-2.0
 *
 *  Unless required by applicable law or agreed to in writing, software
 *  distributed under the License is distributed on an "AS IS" BASIS,
 *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *  See the License for the specific language governing permissions
 *  and limitations under the License.
 */

package shaishav.com.bebetter.data.models

import android.content.ContentValues
import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import shaishav.com.bebetter.data.contracts.GoalContract

/**
 * Created by shaishav.gandhi on 1/5/18.
 */
@Parcelize data class Goal(val id: Long, val date: Long, val goal: Long): Parcelable {

  fun toContentValues(): ContentValues {
    val contentValues = ContentValues()
    contentValues.put(GoalContract.COLUMN_GOAL, goal)
    contentValues.put(GoalContract.COLUMN_DATE, date)
    return contentValues
  }

}